monai/transforms/utility/dictionary.py

```diff
@@ -403,7 +403,10 @@ def __call__(self, data):
         for key, new_key in zip(self.keys * self.times, self.names):
             if new_key in d:
                 raise KeyError(f"Key {new_key} already exists in data.")
-            d[new_key] = copy.deepcopy(d[key])
+            if type(d[key]) == torch.Tensor:
```

Comment:
this could be isinstance torch.Tensor? the type == comparison is not good for subclasses

Reply:
Thanks for your suggestion, let me update it.